test/docs/testplans/vnet_to_vnet.md

```diff
@@ -0,0 +1,58 @@
+# **VNET to VNET**
+
+## Overview
+
+This test plan corresponds to the phase where standardized, automated test cases 
+will be scripted to verify Data plane Testing Stage 3 as outlined in [dash test maturity stages](../dash-test-maturity-stages.md#data-plane-testing-stage-3-dut-configuration-via-sai-thrift). 
+
+In these tests we want to leverage the snappi API to test some of the functional aspects of DUT. 
+By verifying and validating the scaling aspects of the switches (hardware/BMv2) 
+we are able to empirically baseline the scaling values. Additionally, we also 
+want to check some of the relevant counters to features like IPv4, IPv6, TCP, 
+UDP, common traffic stats etc.
+
+### Scope
+
+These test cases are targeted for testing the feature functionality of DASH implementation. The purpose of these test cases are to verify inbound/outbound/bidirectional traffic streams originating and terminating at the traffic generator. We plan to use the [OTG](https://github.com/open-traffic-generator) (Open traffic generator model) by leveraging the snappi APIs. 
+
+### Testbed
+
+The generic testbed setup may consist of two OTG connected to two ports of a switch (hardware-based or BMv2 software switch). 
+In this specific testbed setup we will use [Ixia-c](https://github.com/open-traffic-generator/ixia-c) which is a modern, powerful, and API-driven open traffic generator model-based traffic tool that will generate traffic to observe the behavior of the device under test (DUT).
+
+### Testbed Topology 
+
+The testbed is setup to emulate a VNET to VNET scenario and to test the mechanisms of VM to VM communication in VNET as outlined [here](https://github.com/Azure/DASH/blob/main/documentation/vnet2vnet-service/vnet-to-vnet-service.md).
+
+
+
+To emulate each VM in a VNET an Ixia-c port will be utilized. The BMv2 will be the DUT (This can later be replaced by a hardware-based appliance that connects to physical ports of a traffic generator)
+
+![ixia-c.drawio](../../images/ixia-c.drawio.svg)
```

Comment:
Why not use https://github.com/Azure/DASH/blob/main/test/third-party/traffic_gen/deployment/ixia-c.drawio.svg? It's actually more accurate.

Reply:
Updated the image as per above recommendation